src/new/wallet.js

```diff
@@ -1474,6 +1474,7 @@ class HathorWallet extends EventEmitter {
     if (many) {
       return utxos;
     }
+    return null;
```

Comment:
Why not an empty array?

Reply:
There are some places that expect a `null` response if no authority utxo is found, if they require a list (empty if none are found) they would indicate this with the option `many: true` which makes the response always be an array.

This is an example (and the reason i found this bug) of a place that expects `null`
https://github.com/HathorNetwork/hathor-wallet-lib/blob/3ed53d0dbff27d87da6e267bf16187c5594a32ba/src/new/wallet.js#L609-L610